src/test/kotlin/no/nav/tilleggsstonader/sak/infrastruktur/database/repository/BehandlingRepositoryFake.kt

```diff
@@ -0,0 +1,94 @@
+package no.nav.tilleggsstonader.sak.infrastruktur.database.repository
+
+import no.nav.tilleggsstonader.kontrakter.felles.Stønadstype
+import no.nav.tilleggsstonader.sak.behandling.domain.Behandling
+import no.nav.tilleggsstonader.sak.behandling.domain.BehandlingRepository
+import no.nav.tilleggsstonader.sak.behandling.domain.BehandlingResultat
+import no.nav.tilleggsstonader.sak.behandling.domain.BehandlingStatus
+import no.nav.tilleggsstonader.sak.behandling.domain.EksternId
+import no.nav.tilleggsstonader.sak.behandling.domain.Saksbehandling
+import no.nav.tilleggsstonader.sak.felles.domain.BehandlingId
+import no.nav.tilleggsstonader.sak.felles.domain.FagsakId
+import no.nav.tilleggsstonader.sak.felles.domain.FagsakPersonId
+import no.nav.tilleggsstonader.sak.util.saksbehandling
+
+class BehandlingRepositoryFake :
+    DummyRepository<Behandling, BehandlingId>({ it.id }),
+    BehandlingRepository {
+    override fun findByFagsakId(fagsakId: FagsakId): List<Behandling> {
+        TODO("Not yet implemented")
+    }
```

Comment:
Hva tenker dere om mange funksjoner som ikke er implementert for Fake? Jeg trenger kun `finnSaksbehandling()`, men lagde en Fake for å følge kodestil i `LæremidlerBeregnYtelseStegStepDefinitions.kt`

Reply:
Tenker det er helt OK